build.go

```diff
@@ -1,11 +1,15 @@
 package main
 
+import  (
+	"strings"
+)
+
 func build(cmd Commander, inputs Inputs) error {
 	args := []string{
 		"build",
 		"--file", inputs.Dockerfile,
 	}
-
+	args = strings.toLower(args)
```

Comment:
`args` is a slice of string, you can't call `strings.toLower` with a slice. In your case, you should use `with.repository` argument to make sure tag name is lowercase. Or you should call `strings.toLower` before line 22 to modify the tag.